mod/admin.php

```diff
@@ -2123,12 +2092,10 @@ function __get_theme_admin_form(App $a, $theme)
 	// reload active themes
 	if (x($_GET, "a") && $_GET['a'] == "r") {
 		check_form_security_token_redirectOnErr(System::baseUrl() . '/admin/themes', 'admin_themes', 't');
-		if ($themes) {
-			foreach ($themes as $th) {
-				if ($th['allowed']) {
-					uninstall_theme($th['name']);
-					install_theme($th['name']);
-				}
+		foreach ($themes as $th) {
```

Comment:
I haven't checked: Can we be sure that ```$themes``` is always an array?

Reply:
Defined as such on line 1963, populated on line 1982.